Lib/test/test_future.py

```diff
@@ -74,6 +76,16 @@ def test_badfuture10(self):
             from test import badsyntax_future10
         self.check_syntax_error(cm.exception, "badsyntax_future10", 3)
 
+    def test_ensure_flags_dont_clash(self):
+        flags = [
+            getattr(__future__, future).compiler_flag
+            for future in __future__.all_feature_names
+        ]
+        flags.extend(
+            (ast.PyCF_ALLOW_TOP_LEVEL_AWAIT, ast.PyCF_ONLY_AST, ast.PyCF_TYPE_COMMENTS)
```

Comment:
I think this can fail again if we add more constants in the `ast` module since we can forget to add them here, subverting the purpose of this test.